app/eg020_sms_authentication.py

```diff
@@ -0,0 +1,159 @@
+                        document_id="1",  # a label used to reference the doc
+                        file_extension="pdf",  # many different document types are accepted
+                        name="Lorem"  # can be different from actual file name
+            )
+            envelope_definition.documents = [document1]
+            envelope_definition.status = args['envelope_args']['status']            
+            signer1 = Signer(
+                email=args["envelope_args"]["signer_email"], # represents your {signer_email}
+                name=args["envelope_args"]["signer_name"], # represents your {signer_name}
+                sms_authentication={ "senderProvidedNumbers": ['415-555-1212']},
```

Comment:
The C# and Node examples get this number from the user in a textbox. We should try to do that as well...

Reply:
on the leadup to momentum I did that but was asked to scrap it so we could have consistency with the examples... I moved the input box onto the view portion and did the same for eg021 and 019 (instead of having it be a phone number it's a string of text to use for the access code.